weston-ivi-shell/src/ivi-controller.c

```diff
@@ -1889,13 +1905,18 @@ surface_event_configure(struct wl_listener *listener, void *data)
     uint32_t surface_id;
     struct weston_surface *w_surface;
 
+    w_surface = lyt->surface_get_weston_surface(layout_surface);
+    /* ivi_shell only emit the surface configured event when
+       it have the content and new size of buffer. Otherwise,
+       when the id-agen set the surface id for surface. */
```

Comment:
Grammar correction:
emit -> emits

When it have -> when it has

correction id-agen -> id-agent